lib/schema/qa3_user_ids.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
You might as well go ahead and check in similar files for all the environments. That way if this is deployed anywhere it will still function, but effectively act the same and use Jellyfish.